test_unstructured_ingest/test-ingest-against-api.sh

```diff
@@ -2,22 +2,18 @@
 
 set -e
 
-SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
```

Comment:
I ran into an issue with this when called from check-X scripts. The updated form does work everywhere, so updated across the board for consistency.